manual_scripts/active_directory/set-password-policy.ps1

```diff
@@ -0,0 +1,6 @@
+# Policy taken from Datamart prod
+
+Set-ADDefaultDomainPasswordPolicy -Identity dluhcdata.local -ComplexityEnabled $True `
+ -LockoutDuration 00:30:00 -LockoutObservationWindow 00:30:00 `
+ -ReversibleEncryptionEnabled $False -MinPasswordAge 1.00:00:00 -MaxPasswordAge 90.00:00:00 `
```

Comment:
90 days is so short 🙁

Reply:
Yeah and PasswordHistoryCount 24 is definitely too many to remember...

But I don't feel like asking for a change here.